app/assets/javascripts/actions/uploads_actions.js

```diff
@@ -109,6 +109,49 @@ export const setUploadViewerMetadata = (upload) => dispatch => {
   );
 };
 
+const fetchUploadPageViews = (articleList) => {
+  const viewPerArticle = [];
+  // set the start date to 60 days form today
+  const rawDate = new Date();
+  rawDate.setDate(rawDate.getDate() - 60);
+  const startDate = new Date(rawDate).toJSON().slice(0, 10).replace(/-/g, '');
+  const endDate = new Date().toJSON().slice(0, 10).replace(/-/g, '');
```

Comment:
This code is not self-explanatory, and is also repeated. One way to handle this is to extract a function like `pageViewDateString`, and add a comment to note the target format of YYYYMMDD.